scripts/install-elastio.sh

```diff
@@ -85,6 +85,11 @@ deb_ubu_install()
     wget $repo_url/deb/Debian/${debian_ver}/pool/$pkg
     dpkg -i $pkg && rm -f $pkg
     apt-get update
+    # Maybe new kernel was installed recently but machine was not yet booted into it.
+    # In this case dkms will install as dependency linux-headers-[latest kernel version].
+    # But we need to have driver built and loaded right now without reboot, that's why
+    # installing linux-headers package for the current kernel.
```

Comment:
This seems like a good fix, but is it really Debian/Ubuntu specific?  Couldn't this same scenario happen with other distros?

Reply:
We are already installing `kernel-devel-$(uname -r)` and `kernel-devel` packages on CentOS/Fedora

https://github.com/elastio/elastio-stack/blob/bafbc79580f8893afd110f1038be1476b4d0c55d/scripts/install-elastio.sh#L9-L19